proposals/0000-arkworks-integration.md

```diff
@@ -0,0 +1,95 @@
+---
+Title: Arkworks Integration
```

Comment:
Should be something like "Eliptic Curve Host Functions"

Reply:
Done